README.md

```diff
@@ -112,6 +115,8 @@ module.exports = {
     env: {
       ROOT_URL: 'http://app.com',
       MONGO_URL: 'mongodb://localhost/meteor'
+      PORT: 80, // (optional) defaults to 80
+      BIND: '0.0.0.0' // (optional) defaults to 0.0.0.0
```

Comment:
I've seen you leaved like `config.env.PORT || 80` in modules/meteor/index.js line 131, is that intended?